mediawiki/semantic-media-wiki/CVE-2022-48614.yaml

```diff
@@ -0,0 +1,8 @@
+title: 'Cross-site Scripting in Semantic MediaWiki'
+link: 'https://nvd.nist.gov/vuln/detail/CVE-2022-48614'
```

Comment:
I would link to https://www.semantic-mediawiki.org/wiki/Semantic_MediaWiki_4.0.2 here

Reply:
I agree

Reply:
There are already some links in the repository which are not working. I think more reliable are links to NVD.

Maybe the solution would be to allow for more links and/or references? What do you think?